versioned_docs/version-8.2/self-managed/platform-deployment/helm-kubernetes/upgrade.md

```diff
@@ -120,7 +120,77 @@ Camunda Platform v8.2 uses Keycloak v19 which depends on PostgreSQL v15. That is
 
 **Method 1: Upgrade the database schema to work with PostgreSQL v15**
```

Comment:
We probably need to indicate that this step is done before the actual chart upgrade.

Reply:
I agree with this, though I'm not sure how I'd go about adding this in. these upgrade docs are structured in kindof a reactive way. If you encounter this issue, do this. rather than a process someone would follow from beginning to end.  Maybe we can just include a command as a step 11 that says `helm upgrade [...]`.

Thoughts?

Reply:
I added a step at the end that says to continue with the helm upgrade. Let me know if this works.

Reply:
You're right @aabouzaid , although I'm not sure where best to note that. Would you mind adding a commit if you have any ideas? 

I wrote this snippet that works syntactically, but I'm not sure if it's what you want.
```
:::caution Extract secrets
Make sure to follow instructions for [Secrets extraction](#secrets-extraction) prior to running helm upgrade.
:::
```

In the mean time, I can revert that 10th step that suggests that the user runs `helm upgrade`.

Reply:
I took a look and I like the changes. Thank you @aabouzaid !